crates/http/src/init.rs

```diff
@@ -0,0 +1,17 @@
+use axum::{response::Html, routing::get, Router};
+
+pub async fn init() {
+    tokio::spawn(async {
+        let routes = Router::new().route("/", get(Html("Hello world!")));
+
+        let addr = std::env::var("IRON_HTTP_SERVER_ENDPOINT")
```

Comment:
can you also change the ws env name to `IRON_WS_...` to match?

Reply:
Yup, not a problem